features/step_definitions/register_page_steps.rb

```diff
@@ -0,0 +1,60 @@
+Given(/^there is a register page$/) do
+  visit '/signup'
+end
+
+When(/^I visit the register page$/) do
+  visit '/signup'
+end
+
+Then(/^I would like to see a form for registration$/) do
+  expect(page).to have_selector 'form'
+end
+
+Then(/^I would like to see a text field for user name$/) do
+  expect(page).to have_field 'Username'
+end
+
+Then(/^I would like to see a text field for password$/) do
+  expect(page).to have_field 'Password'
+end
+
+Then(/^I would like to see a text field for password confirmation$/) do
+  expect(page).to have_field 'Password confirmation'
+end
+
+Then(/^I would like to see a register button$/) do
+  expect(page).to have_selector(:link_or_button, "Sign up")
+end
+
+Given(/^username and password has been entered$/) do
+  visit '/signup'
+  fill_in 'username', :with => 'alfred'
+  fill_in 'password', :with => 'whatever'
+  fill_in 'passwordconfirmation', :with => 'whatever'
```

Comment:
We try not to use `hash rockets`. That is an old syntax. 
```
fill_in 'username', with: 'alfred'
```
Also, please use `snake_case` to separate words. Like `passwordconfirmation` should be `password_confirmation` and `username` should be `user_name` It's a convention that aim to solve the readability problem.